### PR TITLE
Feature/initial implementation

### PR DESCRIPTION
**Pull request by jacob-andersson**

### CODE REVIEW

File: Pod/Pod/ViewController.m [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+#pragma mark - Getters
+
+- (HYPMediaPageViewController *)mediaPageViewController
+{
+    if (_mediaPageViewController) return _mediaPageViewController;
+
+    _mediaPageViewController = [HYPMediaPageViewController new];
+    _mediaPageViewController.view.translatesAutoresizingMaskIntoConstraints = NO;
```
Comment (3lvis) [inline Pod/Pod/ViewController.m]: Any reason why this isn't in the `HYPMediaPageViewController` initializer?

File: Pod/Pod/ViewController.m [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+    [_mediaPageViewController setImageURLStrings:@[@"http://lorempixel.com/640/640/sports",
+                                                   @"http://lorempixel.com/g/640/640/transport",
+                                                   @"http://lorempixel.com/640/640/food/"]
+                             andPlaceholderImage:[UIImage imageNamed:@"hypPlaceholderImage"]];
+
+    [_mediaPageViewController.view addConstraint:
```
Comment (3lvis) [inline Pod/Pod/ViewController.m]: Any reason why this isn't in the `HYPMediaPageViewController` initializer?

File: Source/HYPMediaPageItemViewController.h [hunk 3]
```diff
@@ -0,0 +1,8 @@
+@import UIKit;
+
+@interface HYPMediaPageItemViewController : UIViewController
+
+@property (nonatomic, strong) UIImageView *imageView;
```
Comment (3lvis) [inline Source/HYPMediaPageItemViewController.h:5]: Please drop the `strong` is inferred now.

File: Pod/Pod/ViewController.m (hunk 1 above)
Comment (jacob-andersson) [inline Pod/Pod/ViewController.m]: It does not have any special initializer, this is just the demo use. Or perhaps I misunderstood you?

**Pull request merged**